src/payment/strategies/paypal-express-payment-strategy.ts

```diff
@@ -56,13 +52,15 @@ export default class PaypalExpressPaymentStrategy extends PaymentStrategy {
 
         if (this._isInContextEnabled() && this._paypalSdk) {
             this._paypalSdk.checkout.closeFlow();
-            this._paypalSdk = null;
+            this._paypalSdk = undefined;
```

Comment:
I guess this might get tricky as it clears the paypalSDK attached to the class, but not from the window. We might be a wider issue with other strategies that use a similar approach.